readme.md

```diff
@@ -240,6 +240,7 @@
 - [Integration](https://github.com/stn1slv/awesome-integration#readme) - Linking together different IT systems (components) to functionally cooperate as a whole.
 - [Node-RED](https://github.com/naimo84/awesome-nodered#readme) - A programming tool for wiring together hardware devices, APIs, and online services.
 - [Low Code](https://github.com/zenitysec/awesome-low-code#readme) - Allowing business professionals to address their needs on their own with little to no coding skills.
+- [Clubhouse](https://github.com/ehsanghaffarii/awesome-clubhouse#readme) - helpful tools, and resources for developers and users.
```

Comment:
> Your entry here should include a short description about the project/theme of the list. It **should not describe the list itself**. Your entry here should include a short description about the project/theme of the list. It should not describe the list itself. **The first character should be uppercase**…

This description both describes the list, instead of the project, the first character should also start in a capital.